opennebula/data_opennebula_user.go

```diff
@@ -19,7 +19,7 @@ func dataOpennebulaUser() *schema.Resource {
 			},
 			"password": {
```

Comment:
it is a complete non sense to ask for password even in an optional way. Please remove the password attribute from this data source.

Reply:
fixed